packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -256,18 +264,19 @@ class _CupertinoSliderRenderObjectWidget extends LeafRenderObjectWidget {
   final double value;
   final int divisions;
   final Color activeColor;
+  final Color thumbColor;
   final ValueChanged<double> onChanged;
   final ValueChanged<double> onChangeStart;
   final ValueChanged<double> onChangeEnd;
   final TickerProvider vsync;
 
-
   @override
   _RenderCupertinoSlider createRenderObject(BuildContext context) {
     return _RenderCupertinoSlider(
       value: value,
       divisions: divisions,
       activeColor: activeColor,
+      thumbColor: thumbColor,
```

Comment:
```suggestion
      thumbColor: CupertinoDynamicColor.resolve(thumbColor, context),
```
`thumbColor` needs to be resolved in case it's a [CupertinoDynamicColor](https://master-api.flutter.dev/flutter/cupertino/CupertinoDynamicColor-class.html).